pydantic_graph/pydantic_graph/_utils.py

```diff
@@ -1,12 +1,13 @@
 from __future__ import annotations as _annotations
 
 import asyncio
-import sys
 import types
 from datetime import datetime, timezone
-from typing import Annotated, Any, TypeVar, Union, get_args, get_origin
```

Comment:
Across the projects, I imported `get_args`/`get_origin`/`get_type_hints` from `typing_inspection`, as it provides backports from newer Python versions.

Reply:
Right